scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -2363,6 +2364,27 @@ class FormatOps(
         }
     }
 
+    private object IfImpl extends Factory {
+      def create(ft: FormatToken, nft: FormatToken)(implicit
+          style: ScalafmtConfig
+      ): Option[OptionalBracesRegion] = {
+        ft.meta.leftOwner match {
+          case t: Term.If =>
+            (t.cond match {
+              case b: Term.Block => Some(true)
+              case _ => None
+            }).map { forceNL =>
+              new OptionalBracesRegion {
+                def owner = Some(t)
+                def splits = Some(getSplits(ft, t.cond, forceNL))
+                def rightBrace = blockLast(t.cond)
+              }
+            }
```

Comment:
```suggestion
            t.cond match {
              case b: Term.Block =>
                new OptionalBracesRegion {
                  def owner = Some(t)
                  def splits = Some(getSplits(ft, b, true))
                  def rightBrace = blockLast(b)
                }
              case _ => None
            }
```

Reply:
do we need to check that the block contains more than one statement?

Reply:
We don't treat it as block in the parser if it's a single indented statement. That's in line how Scala 3 parser works.

Reply:
Changed, but added `Some()` around.